packages/ringcentral-widgets/components/Modal/styles.scss

```diff
@@ -26,6 +26,7 @@
 }
 
 .modal {
```

Comment:
em..we the modal has the`transform` property to reset the coordinate system， practically we do not have to, but it would be better according to experience since not everyone know this bug in the W3C spec